textscreen/txt_sdl.c

```diff
@@ -262,9 +262,24 @@ int TXT_Init(void)
 
     if (TXT_SDLWindow == NULL)
     {
+        int w, h;
+
+        if (font == &normal_font || font == &highdpi_font)
+        {
+            w = 3 * screen_image_w / 2;
+            h = 3 * screen_image_h / 2;
+        }
+        else
+        {
+           w = screen_image_w;
+           h = screen_image_h;
+        }
+        flags |= SDL_WINDOW_RESIZABLE;
```

Comment:
This doesn't affect ENDOOM, right?

Reply:
No, for ENDOOM we use the main game window, which is already resizable.

Reply:
The ENDOOM screen uses the same window and renderer as the main game, so the same settings wrt aspect ratio correction and nearest scaling apply. The rounded edges come from the font itself, the high resolution variant features them.

Reply:
> It also looks like to me that it's been scaled using sai/supersai/eagle/supereagle filter (the text shows some "roundness").

No, it's linear scaling on the GPU.

> Speaking of ENDOOM, shouldn't it benefit from aspect-ratio correction?

There are the same problems as with the setup tool, raster fonts will not look good with scaling not in a 1:1 ratio. I was thinking about using ttf fonts like [Flexi VGA 2.0](https://int10h.org/blog/2020/12/more-font-updates-oldschool-pack-flexi-vga/). DosBox-X has ttf mode, looks interesting.

Reply:
> so the same settings wrt aspect ratio correction and nearest scaling apply

Actually no, it's a 1:1 linear scaling 🤓 I didn't like the nearest scaling for ENDOOM, and some time ago we decided to set up its rendering differently.

Reply:
Yes, of course, I knew that. :neckbeard: No, seriously, we changed so much that I lost track of some details.